packages/frontend/amp/components/topMeta/SeriesLink.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { headline } from '@guardian/pasteup/typography';
+import { css } from 'emotion';
+import { pillarPalette } from '@frontend/lib/pillars';
+
+const seriesStyle = (pillar: Pillar) => css`
+    color: ${pillarPalette[pillar].main};
+    ${headline(2)};
+    font-weight: 900;
+    text-decoration: none;
+    margin-top: 10px;
+    display: block;
+`;
+
+// Returns a series link if possible, and attempt to return a section link as a fallback if provided
+export const SeriesLink: React.SFC<{
+    baseURL: string;
+    tags: TagType[];
+    fallbackToSection: boolean;
+    sectionLabel?: string; // required for fallback only
+    sectionUrl?: string; // required for fallback only
+    pillar: Pillar;
+}> = ({
+    baseURL,
+    tags,
+    fallbackToSection,
+    sectionLabel,
+    sectionUrl,
+    pillar,
+}) => {
+    const tag = tags.find(t => t.type === 'Blog' || t.type === 'Series');
```

Comment:
Do we need to use the filterForTagsOfType function in tag-utils? I'm not entirely sure of the rationale behind that method though.

Reply:
Hmm. That utility method won't work here as it only takes one type... I actually think refactoring the util to just a find would be better, however its currently filtering and returning all tags of type whereas this one just returns the first one... It's only used once so I think this refactor would fine but in another PR as it touches Article.